### PR TITLE
Fix AdBlock blocking share icon

### PR DESCRIPTION
**Pull request by jancborchardt**
Same as https://github.com/nextcloud/server/pull/4337, but for the share icons in the Calendar.

Please review @kumzugloom @Pommesumdreher @MariusBluem @Bullnados @GetBoz @georgehrke @MariusBluem :)

### CODE REVIEW

**Comment by ryanbr** [comment]
Given the domains for this tool changes, no way to cover all instances. If nextcloud has changed the element from icon-share to icon-shared? does that fix it?

File: templates/part.calendarlist.item.php [hunk 1]
```diff
@@ -49,7 +49,7 @@
 		  ng-class="{'withitems': item.calendar.isShared() || item.calendar.isPublished() }">
 		<span
 			class="calendarlist-icon share permanent"
-			ng-class="{'icon-shared': item.calendar.isShared() && !item.calendar.isPublished(), 'icon-public': item.calendar.isPublished(), 'icon-share': !item.calendar.isShared() && !item.calendar.isPublished()}"
+			ng-class="{'icon-shared': item.calendar.isShared() && !item.calendar.isPublished(), 'icon-public': item.calendar.isPublished(), 'icon-shared': !item.calendar.isShared() && !item.calendar.isPublished()}"
```
Comment (nickvergessen) [inline templates/part.calendarlist.item.php]: now the key exists twice

Comment (jancborchardt) [inline templates/part.calendarlist.item.php]: @georgehrke could you please look into this? The images share.svg and shared.svg are exactly the same. They if shared or not should only differ in opacity of the icon, like in Files.

**Comment by jancborchardt** [comment]
@georgehrke @raghunayyar can you check this out so it’s ready for Nextcloud 12? :)

**Comment by jancborchardt** [comment]
Any review please @kumzugloom @Pommesumdreher @MariusBluem @Bullnados @GetBoz @georgehrke @MariusBluem?

**Comment by georgehrke** [comment]
Jan, stay calm :)

The calendar is not affected by today's feature freeze. I will merge this before we release the next calendar version 

Please excuse my brevity and typos.

**Comment by jancborchardt** [comment]
I know, still it’s good practice do do this as early as possible. ;)

**Comment by georgehrke** [comment]
Fixed the styles:
calendar - nextcloud chromium today at 4 42 14 pm

**Pull request merged**